lib/logic/caregiver/tasks_evaluation/tasks_evaluation_cubit.dart

```diff
@@ -90,7 +90,6 @@ class TasksEvaluationCubit extends ReloadableCubit {
 		}
 		await Future.wait(updates);
 		await sendNotification();
-		emit(TasksEvaluationSubmissionSuccess(_reports));
```

Comment:
Why not?

Reply:
I can see that the `TasksEvaluationSubmissionInProgress` is also not being used at all, should probably be emitted at the start of this method